Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -1015,14 +2151,22 @@ def import_ioc_without_approval(client: Client, file_id, classification, confide
     return "The data was imported successfully."
 
 
-def get_model_list(client: Client, model, limit="50"):
+def get_model_list(client: Client, model, limit="50", page=None, page_size=None):
     """
         Returns list of Threat Model that was specified. By default limit is set to 50 results.
         Possible values for model are : actor, campaign, incident, signature, ttp, vulnerability, tipreport
     """
     # if limit=0 don't put to context
-    params = dict(limit=limit, skip_intelligence="true", skip_associations="true")
-    model_list = client.http_request("GET", F"v1/{model}/", params=params).get('objects', None)
+    if (page and not page_size) or (not page and page_size):
```

Comment:
You have this validation already in a helper function, no? why do we need this duplicate code?